analysis/topEFT/make_1d_quad_plots.py

```diff
@@ -19,7 +19,9 @@ def main():
 
     # Set up the command line parser
     parser = argparse.ArgumentParser()
+    parser.add_argument("-i", "--input-path", default="/store/user/kmohrman/FullProduction/FullR2/UL17/Round1/Batch2/naodOnly_step/v2/nAOD_step_ttHJet_all22WCsStartPtCheckdim6TopMay20GST_run0/NAOD-00000_10769.root", help = "The path the input files should be saved to")
     parser.add_argument("-o", "--output-path", default=".", help = "The path the output files should be saved to")
+    parser.add_argument("-r", "--redirector", default="root://deepthought.crc.nd.edu/", help = "The path the output files should be saved to")
```

Comment:
I think the help string for this argument might be describing a different option?